### ISSUE TITLE
Crowdin.authorize() keep crashing: Channel is unrecoverably broken and will be disposed!

### ISSUE THREAD

**Issue by BillHoo** [crowdin/mobile-sdk-android#54]
**Describe the bug**
Every time when logged into Crowdin from `Crowdin.authorize()`, App crashes.

**To Reproduce**

1. Follow the official instructions to setup Crowdin SDK, confirm that `Crowdin.init()` works good.
1. Add `Crowdin.authorize()` to MainActivity's `onCreate()` method
1. Open my App and there will be a Crowdin prompt said need authorize, click `OK` to jump to Crowdin page and input correct password, click login
1. An "In progress ..." page is shown.
1. After 3~4 seconds, App crashed.

**Expected behavior**
After `step #4` above, it should be back to MainActivity and no crash.

**Smartphone (please complete the following information):**
 - Device: Huawei P20
 - OS: Android 10

**Additional context**

- The App is our own working product, just added `Crowdin.authorize()` to MainActivity's `onCreate()` method.
- If I press `Cancel` when prompt, no crash will happen.
- If I press `OK`, then get into the login page, and input nothing, just click `back` button, i could get back to MainActivity normaly.
- When get into the login page, `onPause()` of MainActivity was triggered, but after successful login, the App was crashed and `onResume()` was never triggered.

Based on contexts above, i guess **there's something wrong when authorize succeed and Crowdin trying to resume MainActivity.**

Logs listed below:

```
 E/InputDispatcher: Window handle Window{b240f29 u0 com.myapp.activity/com.myapp.activity.main.MainActivity} has no registered input channel
 E/WindowManager: win=Window{b240f29 u0 com.myapp.activity/com.myapp.activity.main.MainActivity EXITING} destroySurfaces: appStopped=false win.mWindowRemovalAllowed=true win.mRemoveOnExit=true
 ...
 E/WindowManager: win=Window{b1905b9 u0 com.myapp.activity/com.myapp.activity.main.MainActivity} destroySurfaces: appStopped=true win.mWindowRemovalAllowed=false win.mRemoveOnExit=false
 E/WindowManager: win=Window{b296b29 u0 Splash Screen com.myapp.activity EXITING} destroySurfaces: appStopped=false win.mWindowRemovalAllowed=true win.mRemoveOnExit=true
 ...
 E/InputDispatcher: channel 'b27bee1 com.myapp.activity/com.crowdin.platform.auth.AuthActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
 E/InputDispatcher: channel 'b1905b9 com.myapp.activity/com.myapp.activity.main.MainActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
```

**Comment by BillHoo** [comment]
Update:

I guess its the issue of `gson`, cos' I use this one to exclude gson from Crowdin package to avoid conflict with our legacy `andbase.jar`.

I'll get back soon.

```groovy
     implementation ('com.crowdin.platform:mobile-sdk:1.1.4') {
        exclude group: 'com.google.code.gson', module: 'gson'
     }
```



New crash log grabed

```java
2020-06-23 09:24:05.280 29234-29918/com.myapp.activity E/AndroidRuntime: FATAL EXCEPTION: LocalWorkThreadPool-1
    Process: com.myapp.activity, PID: 29234
    java.lang.IllegalAccessError: Method 'com.google.gson.stream.JsonWriter com.google.gson.Gson.newJsonWriter(java.io.Writer)' is inaccessible to class 'retrofit2.converter.gson.GsonRequestBodyConverter' (declaration of 'retrofit2.converter.gson.GsonRequestBodyConverter' appears in /data/app/com.myapp.activity-8tUZA3KoyPogGba3ZBxXIQ==/base.apk)
        at retrofit2.converter.gson.GsonRequestBodyConverter.convert(GsonRequestBodyConverter.java:45)
        at retrofit2.converter.gson.GsonRequestBodyConverter.convert(GsonRequestBodyConverter.java:30)
        at retrofit2.ParameterHandler$Body.apply(ParameterHandler.java:412)
        at retrofit2.RequestFactory.create(RequestFactory.java:117)
        at retrofit2.OkHttpCall.createRawCall(OkHttpCall.java:192)
        at retrofit2.OkHttpCall.execute(OkHttpCall.java:175)
        at retrofit2.DefaultCallAdapterFactory$ExecutorCallbackCall.execute(DefaultCallAdapterFactory.java:104)
        at com.crowdin.platform.auth.AuthActivity$handleCode$1.run(AuthActivity.kt:126)
        at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:929)

```

**Comment by BillHoo** [comment]
Issue was resolved. Sharing the solution here for later user may encounter the issue.

The crash is because I have a legacy dependency named `andbase.jar`, and it contains google GSON as well. While Crowdin SDK depend on `com.google.code.gson:gson:2.8.5`, these two are conflict and will cause a `build time` error.

But the first solution I've tried is to exclude Crowdin's like this

```groovy
  implementation ('com.crowdin.platform:mobile-sdk:1.1.4') {
        exclude group: 'com.google.code.gson', module: 'gson'
  }
```

So the `build time` error was gone and App could run normally. But after the succeed authorize, Crowdin SDK try to invoke GSON (which is now the legacy version from `andbase.jar`), it crashes with the following error at `run time`:

```
java.lang.IllegalAccessError: Method 'com.google.gson.stream.JsonWriter com.google.gson.Gson.newJsonWriter(java.io.Writer)' is inaccessible
```

So the final solution is tricky, I have to keep the legacy `andbase.jar` while exlude any google GSON classes from the jar file:

1. Open `andbase.jar` with any compress tool like 7-Zip or WinRAR
2. Delete all google's folder and classes in it
3. Save `andbase.jar` and re-compile the App
